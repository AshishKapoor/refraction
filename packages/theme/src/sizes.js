// @flow
const sizes = {
  small: '5px',
  regular: '10px',
  large: '20px',
};

export default sizes;
