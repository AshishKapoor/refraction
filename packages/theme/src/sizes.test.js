// @flow
import sizes from './sizes';

it('exposes the right sizes', () => {
  expect(sizes).toMatchSnapshot();
});
