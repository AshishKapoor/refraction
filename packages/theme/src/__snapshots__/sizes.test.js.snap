// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exposes the right sizes 1`] = `
Object {
  "large": "20px",
  "regular": "10px",
  "small": "5px",
}
`;
